### ISSUE TITLE
Xplane Map Enhancement 

### ISSUE THREAD

**Issue by betterpilot** [derekhe/msfs2020-map-enhancement#752]
Using Xplane Map Enhancement - Pro, V2.0.6.0

I used the downloader to download all the files and install. 

When I go to start the service I get this error. 

![Screenshot_38](https://github.com/user-attachments/assets/c04169ed-5032-401a-ac61-2393adf76520)

I turned on the Virtual Disk Service in my task manager and it did not solve the problem. Not sure what to do , please advise.

**Comment by derekhe** [comment]
I think there is a virtual disk driver issue. You can have a look the all the drivers in computer management, turn on the hidden drivers, you will find something about virtual disk, reinstall it.

**Comment by betterpilot** [comment]
I think the files the program is downloading are corrupted which is why the program can't mount the files. 

Please advise on next steps to try.

![Screenshot_36](https://github.com/user-attachments/assets/4583c0fc-7fbf-43b5-bbbd-67eed8f75328)
![Screenshot_35](https://github.com/user-attachments/assets/9c133425-e183-4207-92f2-92d20eccfd6d)

**Comment by derekhe** [comment]
Hello sorry for late reply.

This is caused by download failures and you can see the gsplit reported errors.

You can download the files manually and merge them with the exe files: https://github.com/derekhe/xplane-map-enhancement-release/releases/tag/basepackage-v2

And here is the guide: https://github.com/derekhe/xplane-map-enhancement-release/wiki/Manual-EN#14-manual-install-base-package

**Comment by robvalkyrie** [comment]
Does the Xplane version cause conflicts with MSFS. My MSFS photogrammetry is all of a sudden messed up.

**Comment by robvalkyrie** [comment]
What hosts files should I delete in the drivere/etc folder when flying MSFS so it doesnt conflict

**Comment by derekhe** [comment]
No,they are separate app.





------------------ Original ------------------
From: robvalkyrie ***@***.***&gt;
Date: Fri,Oct 4,2024 6:05 AM
To: derekhe/msfs2020-map-enhancement ***@***.***&gt;
Cc: hesicong ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [derekhe/msfs2020-map-enhancement] Xplane Map Enhancement  (Issue#752)





 
Does the Xplane version cause conflicts with MSFS. My MSFS photogrammetry is all of a sudden messed up.

**Comment by robvalkyrie** [comment]
I looked at my Hosts file. I do not have these files. Do they only appear when its running.
Also, are are you saying that you do not think that’s what was messing with my MSFS issues with imagery problems. Others have said that it might mess with Host file? I reinstalled MSFS and it’s back to normal, afraid to start my Xplane Map Enhancement..lol. Don’t want to go through installed MSFS again, ugg..lol. Maybe my issue was something else. I guess what I am asking is there ANY scenario that Xplane Map Enhancement could be causing the issues with MSFS?
I appreciate your time if you respond. The Flight Sim Community is the best and I appreciate all your work. People like you are simply amazing to me and I am beyond grateful.
